Userland/Libraries/LibPDF/ColorSpace.cpp

```diff
@@ -120,12 +120,12 @@ NonnullRefPtr<DeviceRGBColorSpace> DeviceRGBColorSpace::the()
     return instance;
 }
 
-PDFErrorOr<ColorOrStyle> DeviceRGBColorSpace::style(ReadonlySpan<Value> arguments) const
+PDFErrorOr<ColorOrStyle> DeviceRGBColorSpace::style(ReadonlySpan<float> arguments) const
 {
     VERIFY(arguments.size() == 3);
-    auto r = static_cast<u8>(arguments[0].to_float() * 255.0f);
-    auto g = static_cast<u8>(arguments[1].to_float() * 255.0f);
-    auto b = static_cast<u8>(arguments[2].to_float() * 255.0f);
+    auto r = static_cast<u8>(arguments[0] * 255.0f);
+    auto g = static_cast<u8>(arguments[1] * 255.0f);
+    auto b = static_cast<u8>(arguments[2] * 255.0f);
```

Comment:
Quick question:
In the previous PR there was a place using `round` before casting
any guess why these are different/which is more correct

Reply:
Rounding is better. These are different because they were written by someone else and because this code path isn't covered by `icc --debug-roundtrip`. We should change it, but not in this PR 🙂